### PR TITLE
Without modules

### PR DESCRIPTION
**Pull request by siseer11**
Did:
- GbCard50 ( gb-card-12 all of them of the styleguide )
- GbCardLabel ( used in almost all the cards , the label down with 3 fields )
- GbFooter
- GbNav
- GbNavAside ( navAsideLink , the links for the gbnavaside )
- GbSwitcher

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        'txt' : 'bookmarks',
+        'icon' : <Bookmarks myClases='nav-aside-icon gb-icon-medium gb-icon-white-opacity-50'/>
+      },
+      {
+        'txt' : 'calendar',
+        'icon' : <Calendar myClases='nav-aside-icon gb-icon-medium gb-icon-white-opacity-50'/>
+      },]
+    }
+    userName='Thomas Schneider'
+    userImageUrl='https://images.unsplash.com/photo-1520466809213-7b9a56adcd45?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ&s=6dd9dc582c677370d110940fda65b992'
```
Comment (biancaaaaa) [inline src/App.js]: do you need this comment? if not, please delete it

File: src/components/gbCard50.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      className="card-image gb-phone-hide"
+      style={{ backgroundImage: `url(${background})` }}
+    />
+    <div className="text-content">
+      <h1
+        className={`card-title gb-text-black gb-title-medium ${
+          type === "full" ? "gb-desktop-text-white gb-desktop-title-large" : ""
+        }`}
+      >
+        Arctic sea ice extent hits record low for winter maximum
```
Comment (biancaaaaa) [inline src/components/gbCard50.js]: Why is there still some text? There should be a prop "title" or sth

File: src/components/gbNavAside.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+/*
+<li className="nav-aside-top-list-item">
+            <a className="nav-aside-link gb-text-white" href="#">
+
+              <h5 className="gb-text-uppercase gb-text-white gb-title-tiny">CHANNELS</h5>
+            </a>
+        </li>
+
+<img className="gb-avatar gb-avatar-medium" src="../images/profile.jpg" alt="avatar">
+        */
```
Comment (biancaaaaa) [inline src/components/gbNavAside.js]: is this comment still needed? if not, please delete

File: src/svgs/svgs.js [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+  <svg className={myClases} id="widgets" viewBox="0 0 50 50" width="100%" height="100%">
+      <title>widgets</title>
+      <g>
+        <path d="M45 23C45 35.1503 35.1503 45 23 45C10.8497 45 1 35.1503 1 23C1 10.8497 10.8497 1 23 1C35.1503 1 45 10.8497 45 23Z" transform="translate(2 1)" strokeWidth="2"></path>
+        <path d="M28 14C28 6.26801 21.732 0 14 0C6.26801 0 0 6.26801 0 14" transform="translate(11 10)" strokeWidth="2" strokeLinecap="round"></path>
+        <path d="M8 4.5C8 6.433 6.433 8 4.5 8C2.567 8 1 6.433 1 4.5C1 2.567 2.567 1 4.5 1C6.433 1 8 2.567 8 4.5Z" transform="translate(25.2324 24.0249) rotate(45)" strokeWidth="2"></path>
+        <path d="M2 5.8V0.136079" transform="translate(31.0461 22.0396) rotate(45)" strokeWidth="2" strokeLinecap="round"></path>
+      </g>
+    </svg>
+)
```
Comment (biancaaaaa) [inline src/svgs/svgs.js]: please split into separate files

File: src/App.js (hunk 1 above)
Comment (siseer11) [reply src/App.js]: Done

File: src/components/gbCard50.js (hunk 2 above)
Comment (siseer11) [reply src/components/gbCard50.js]: Done

File: src/components/gbNavAside.js (hunk 3 above)
Comment (siseer11) [reply src/components/gbNavAside.js]: Done

File: src/svgs/svgs.js (hunk 4 above)
Comment (siseer11) [reply src/svgs/svgs.js]: Done

**Pull request merged**